h/accounts/schemas.py

```diff
@@ -246,11 +246,11 @@ class ResetPasswordSchema(CSRFSchema):
     # call it `user` because when validated, it will return a `User` object.
     user = colander.SchemaNode(
         ResetCode(),
-        title=_('Your reset code:'),
+        title=_('Reset code'),
```

Comment:
So you've changed the form field titles on the login and reset password forms to be:

- terser
- colon-less

But you haven't done the same for the "register" or "forgot password" forms. I'm guessing this is an oversight...?